src/main/java/dev/mja00/villagerLobotimizer/LobotomizeStorage.java

```diff
@@ -23,15 +25,21 @@ public class LobotomizeStorage {
     private static final EnumSet<Material> IMPASSABLE_REGULAR_TALL;
     private final VillagerLobotimizer plugin;
     private final NamespacedKey key;
-    private final Set<Villager> activeVillagers = new LinkedHashSet<>(128);
-    private final Set<Villager> inactiveVillagers = new LinkedHashSet<>(128);
+    private final Set<Villager> activeVillagers = Collections.newSetFromMap(new ConcurrentHashMap<>(128));
+    private final Set<Villager> inactiveVillagers = Collections.newSetFromMap(new ConcurrentHashMap<>(128));
     private long checkInterval;
     private long inactiveCheckInterval;
     private long restockInterval;
     private boolean onlyProfessions;
     private boolean lobotomizePassengers;
     private Sound restockSound;
     private Logger logger;
+    private boolean blockChangeDetectionEnabled;
+    private boolean tpsBasedDetection;
+    private double tpsThreshold;
+    private final Map<Chunk, Long> changedChunks = new ConcurrentHashMap<>();
+    private int priorityCheckRadius; // Radius to check for villagers around block changes
```

Comment:
This seems unused, was it planned to be used and forgotten in favor of reloading the entire chunk?